client/fingerprint/landlock.go

```diff
@@ -0,0 +1,38 @@
+package fingerprint
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/shoenig/go-landlock"
+)
+
+const (
+	landlockKey = "kernel.landlock"
+)
+
+// LandlockFingerprint is used to fingerprint the kernal landlock feature.
```

Comment:
```suggestion
// LandlockFingerprint is used to fingerprint the kernel landlock feature.
```

I can't tell if you're trying to pollute our pristine source with some BSDism or if it's a typo. :thinking:

Reply:
Not sure how that happened, I even have spellcheck enabled :sweat: